x/qgb/keeper/keeper_data_commitment.go

```diff
@@ -7,11 +7,11 @@ import (
 
 // TODO add unit tests for all the keepers
 
-// GetCurrentDataCommitment creates latest data commitment at current height according to
+// GetCurrentDataCommitment creates the latest data commitment at current height according to
 // the data commitment window specified
 func (k Keeper) GetCurrentDataCommitment(ctx sdk.Context) (types.DataCommitment, error) {
 	beginBlock := uint64(ctx.BlockHeight()) - k.GetDataCommitmentWindowParam(ctx)
-	endBlock := uint64(ctx.BlockHeight())
+	endBlock := uint64(ctx.BlockHeight()) - 1
```

Comment:
[optional nit]
perhaps add a line of reasoning here

Reply:
[f8ebd4b](https://github.com/celestiaorg/celestia-app/pull/1217/commits/f8ebd4b1772682c3233982fb4b5b9160f2c6b046) is this good?